### ISSUE TITLE
Image thumbnails are wanted

### ISSUE THREAD

**Issue by sethyuan** [xyhp915/logseq-assets-plus#2]
Images pasted into Logseq all have the same file name and without a thumbnail there is no way to know which is the image I want to use.

<img width="651" alt="image" src="https://github.com/xyhp915/logseq-assets-plus/assets/3410293/d2891958-0b37-4c6d-bc0a-2eac48d49c23">

**Comment by jaroet** [comment]
Yes ... thumbnails would be great but not only for images. All documents that a preview-able (if that is a correct word) should do so. Nice plugin ... 👍

**Comment by duiliuliu** [comment]
There is a similar project issue, can watch it together
https://github.com/sosuisen/logseq-cardbox/issues/4
